philistine/mne/io.py

```diff
@@ -199,4 +201,4 @@ def _rename_bv(vhdr_fname):
 def _extract_bv_segments(vmrk_fname):
     ''' Extract segments from BrainVision VMRK file)
     '''
-    pass
\ No newline at end of file
+    pass
```

Comment:
be careful with blindly changing whether a file ends on a newline (this is probably a setting in your editor). Although not typically an issue today, there is a fair amount of older software (including EEG software!) that is quite sensitive to the presence or absence of a final newline.

Reply:
Indeed, my editor ([atom](atom.io)) settings insert a newline. That's interesting, I haven't given it much thought, and apparently that's based on some unix convention see [stackoverflow answer](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795).